scripts/migration/populate_github_branch.py

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env python
+# Command to run: python manage.py shell < scripts/migration/populate_github_branch.py
+"""
+Populate existing challenges with github_branch="challenge" for backward compatibility.
+
+This script should be run after the migration to ensure all existing challenges
+have the github_branch field populated with the default value.
+"""
+
+import traceback
+
+from challenges.models import Challenge
+from django.db import models
+
+
+def populate_github_branch_fields():
+    """
+    Populate existing challenges with empty github_branch fields to use "challenge" as default.
+    """
+    print("Starting github_branch field population...")
+
+    challenges_to_update = (
+        Challenge.objects.filter(github_repository__isnull=False)
+        .exclude(github_repository="")
+        .filter(
+            models.Q(github_branch__isnull=True) | models.Q(github_branch="")
+        )
```

Comment:
If we already have this script to populate and back-fill, why do we need to the populate/reverse populate methods at all?